# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/cron/syslog_atrun.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./misc.c.orig	Sun Jun 10 14:55:07 2001
+++ ./misc.c	Sun Jun 10 14:54:32 2001
@@ -531,8 +531,11 @@
 	openlog(ProgramName, LOG_PID);
 # endif
 	  
-	syslog(LOG_INFO, "(%s) %s (%s)", username, event, detail);
-
+ 	/* HACK: Don't create syslog messages for atrun */
+ 	if ( strcmp(username, "root") || strcmp(event, "CMD") ||
+ 	     strcmp(detail, "test -x /usr/lib/atrun && /usr/lib/atrun") )
+ 		syslog(LOG_INFO, "(%s) %s (%s)\n", username, event, detail);
+  
 	closelog();
 #endif /*SYSLOG*/
 
